Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[workspace]
+members = [
+  "frb_codegen",
+  "frb_rust",
+  "frb_macros",
+  "frb_example/pure_dart/rust",
+  "frb_example/pure_dart_multi/rust",
+  "frb_example/with_flutter/rust",
+]
+
+[workspace.package]
+version = "1.47.0"
```

Comment:
What does the version here mean? If we need to keep it in updated, we have to modify justfile

Reply:
This is explained in this [article](https://betterprogramming.pub/workspace-inheritance-in-rust-65d0bb8f9424).
One can define a version in root manifest and reuse it in workspace members crates manifests automatically.

Reply:
<img width="730" alt="Screen Shot 2565-09-23 at 10 55 41" src="https://user-images.githubusercontent.com/21016014/191889569-38285056-a8df-4ae5-94af-6de4c5f87e53.png">

Reply:
Btw why is your diff blue not green ;)

Reply:
Then we need to modify `justfile`

Reply:
first modif in 7069afee7b81fbaf2c5d5b76bf04c60b42a5cc86